.ado/templates/react-native-init.yml

```diff
@@ -62,7 +62,20 @@ steps:
     displayName: Add npm user to verdaccio
 
   - script: |
-      npx --no-install beachball publish --branch origin/$(System.PullRequest.TargetBranch) --no-push --registry http://localhost:4873 --yes --access public --changehint "Run `yarn change` from root of repo to generate a change file."
+      echo BeachBallBranchName =  $(System.PullRequest.SourceBranchName)
+      echo ##vso[task.setvariable variable=BeachBallBranchName]$(System.PullRequest.SourceBranchName)
+    displayName: Set BeachBall Branch name for PR
+    condition: eq('$(Build.Reason)', 'PullRequest')
+
+  - script: |
+      set
+      echo BeachBallBranchName =  $(Build.SourceBranch)
+      echo ##vso[task.setvariable variable=BeachBallBranchName]$(Build.SourceBranch)
+    displayName: Set BeachBall Branch name for CI
+    condition: ne('$(Build.Reason)', 'PullRequest')
```

Comment:
Consider extracting to a template if used in multiple places. (also `set` in one script but not other)

Reply:
yea was for debugging. Will address tomorrow.